src/app/pages/entradas/ver-comprobante/ver-comprobante.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnInit } from '@angular/core';
+
```

Comment:
Constantes... Dejarlas después de los `import` para que queden "quemadas" en un único lado
```suggestion

export const MVTO_DB = 344;
export const MVTO_CR = 345;

```